### ISSUE TITLE
Polymer Build: js-transform: Unable to optimize...

### ISSUE THREAD

**Issue by busynest** [Polymer/tools#3389]
For the past few weeks, all new project do not build properly. must have come in the latest updates.

info:   (es5-bundled) Building...
warn:   js-transform: Unable to optimize C:\Users\BusyNest\Desktop\dev\tp-web\src\components\my-app.js
{ err: 'Cannot read property \'isIfStatement\' of null' }
info:   (es5-bundled) Build complete!

**Comment by Westbrook** [comment]
You can see this occur by taking a clean clone of `pwa-starter-kit` switching to `template-typescript` and running `yarn build:static`.

After a quick review I'm not seeing any issues with the actual code output, but it's certainly an annoyance.

**Comment by usergenic** [comment]
https://github.com/Polymer/tools/pull/3388/ should fix this.

**Comment by usergenic** [comment]
tl;dr - it's a problem with `babel-preset-minify` so we disable a few plugins it uses to work around the issue.

**Comment by usergenic** [comment]
Published polymer-cli@1.9.8 and polymer-build@3.1.3 fixing this issue.
